packages/jest-mock-apollo/src/MockApolloLink.ts

```diff
@@ -79,7 +77,7 @@ export default class MockApolloLink extends ApolloLink {
             ),
           };
         } catch (error) {
-          result = error;
+          result = normalizeError(error);
```

Comment:
would it make more sense to just re`throw` the normalized error here? Then the test would actually fail regardless of assertions. It feels a bit weird to me to require users to look at a runtime value and in their component to see they messed up their mock format.

Reply:
I sorta tried to leave general functionality as intact as possible. I didn't know if there's a use case where a consumer would want to test their component's error handling, in which case it might be better to let them explicitly define their pass-fail behaviour through assertions? Would maybe just logging the error suffice?